### ISSUE TITLE
Would be good to add an "alternatives" section with ducks?

### ISSUE THREAD

**Issue by Hattshire** [Arinerron/re-lmgtfy#11]
Just thought that would be funny that from one day to another Google looses half of their traffic, and maybe duckduckgo going down for a bit :smiling_imp: 
Here the link:
[Let me duck that for you](https://lmddgtfy.net)

**Comment by Arinerron** [comment]
Good idea. Lol, ngl I'm surprised that people are still looking at this project, it's so old and poorly written 😅

**Comment by Hattshire** [comment]
Tag magic :3
